website/content/en/highlights/2022-02-08-disk-buffer-v2-beta.md

```diff
@@ -0,0 +1,81 @@
+---
+date: "2022-02-08"
+title: "Beta release of Vector's new disk buffer implementation"
+description: "More consistent performance and resource usage for disk buffers"
+authors: ["tobz"]
+pr_numbers: [9476]
+release: "0.20.0"
+hide_on_release_notes: false
+badges:
+  type: enhancement
+---
+
+As the first major milestone in our work to improve the buffering support in Vector, we're excited
+to announce the beta release of a new disk buffer implementation, affectionately named "disk buffer
+v2".
+
```

Comment:
This article is long enough (which is great!) that it could probably benefit from a summary / outline at the top of what people can expect to find below.

I'd also include a lede like:

> Vector's new beta disk buffers are faster, more consistent, and use less resources. With 0.20.0 you can opt into these new disk buffers via a simple configuration update. We need your help to try them out and give us feedback as this new feature stabilises and becomes Vector's default buffer implementation.

To motivate why users should even care.

I'd also add a call to action here (with a link down to the details below about how to do it) to try it out and give us feedback.

Reply:
I've updated the above: what do you think?